### PR TITLE
remove path-matcher filter

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

### CODE REVIEW

File: api/envoy/v9/http/path_matcher/config.proto [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-syntax = "proto3";
-
-package espv2.api.envoy.v9.http.path_matcher;
-
-import "api/envoy/v9/http/common/base.proto";
-import "validate/validate.proto";
-
-message PathMatcherRule {
-  // The pattern used to match a HTTP request.
-  espv2.api.envoy.v9.http.common.Pattern pattern = 1;
```
Comment (TAOXUY) [inline api/envoy/v9/http/path_matcher/config.proto:24]: Also delete this type in common.proto

Comment (qiwzhang) [reply api/envoy/v9/http/path_matcher/config.proto:24]: Done

**Review by nareddyt — APPROVED** [review]

**Review by TAOXUY — APPROVED** [review]

**Pull request merged**